### ISSUE TITLE
Bug with deleting leading new lines in the block

### ISSUE THREAD

**Issue by Zaczero** [codex-team/editor.js#726]
When you put some extra new lines with Shift+Enter at the very beginning of the block and then try deleting it with Backspace the cursor jumps to the previous block.

![GIF bug preview on the home page](https://i.imgur.com/FTs66Uj.gif)

**Comment by neSpecc** [comment]
Thanks, we will fix this in next releases.

**Comment by hjnilsson** [comment]
Notice the same issue, the following data will trigger the error:

```
{
    "time": 1557230398668,
    "blocks": [
        {
            "type": "header",
            "data": {
                "text": "A heading!",
                "level": 2
            }
        },
        {
            "type": "paragraph",
            "data": {
                "text": "<br>You can't delete the newline above this..."
            }
        }
    ],
    "version": "2.13.0"
}
```

**Comment by gohabereg** [comment]
Resolved by #826
